src/js/common/components/AlertBox.jsx

```diff
@@ -33,7 +33,7 @@ class AlertBox extends React.Component {
 
   render() {
     if (!this.props.isVisible) {
-      return <div aria-live="assertive"/>;
+      return null;
```

Comment:
Meanwhile, there will be a bit of extra top margin in Rx, but it's not too noticeable.